### PR TITLE
Fixes tests failing due to new json dump format

### PR DESCRIPTION
**Pull request by Stagno**
## Technical Description

Integration tests for the StageMerger and StageReordering passes were failing because of the change in the json dump format. In this PR the reference json files are corrected and the tests pass.

## Testing

Run tests with `gtclang-tester-no-codegen.sh`

## Dependencies

This PR depends on MeteoSwiss-APN/dawn#203

### CODE REVIEW

**Comment by Stagno** [comment]
launch pipeline

**Comment by Stagno** [comment]
launch pipeline

File: bundle/CMakeLists.txt [hunk 1]
```diff
@@ -90,8 +90,8 @@ yoda_find_package(
   REQUIRED_VARS dawn_DIR
   ADDITIONAL
     DOWNLOAD_DIR ${YODA_DOWNLOAD_DIR}
-    GIT_REPOSITORY "git@github.com:MeteoSwiss-APN/dawn.git"
-    GIT_TAG "master" 
+    GIT_REPOSITORY "git@github.com:Stagno/dawn.git"
```
Comment (twicki) [inline bundle/CMakeLists.txt]: remember to change this back before we merge

**Comment by twicki** [comment]
this can be merged once all the tests pass

**Comment by Stagno** [comment]
launch pipeline

**Comment by Stagno** [comment]
launch pipeline

**Pull request merged**